### PR TITLE
Augumented tokio::net::Incoming with SuspendableStream

### PR DESCRIPTION
**Pull request by seunlanlege**

### CODE REVIEW

**Comment by seunlanlege** [comment]
@tomaka please approve

**Comment by seunlanlege** [comment]
@tomusdrw please approve

File: server-utils/src/suspendable_stream.rs [hunk 1]
```diff
@@ -0,0 +1,92 @@
+use std::time::{Duration, Instant};
+use tokio_timer::Delay;
+use std::io;
+use core::futures::prelude::*;
+use core::futures::task;
+/// `Incoming` is a stream of incoming sockets
+/// Polling the stream may return a temporary io::Error (for instance if we can't open the connection because of "too many open files" limit)
```
Comment (dvdplm) [inline server-utils/src/suspendable_stream.rs]: Please wrap this line

**Comment by seunlanlege** [comment]
My git skills aren't so good, I'm going to close this PR and open a new one as per @tomusdrw's comment

**Pull request closed without merging**